### PR TITLE
deploying database schemas 

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \ deploying database schemas 
* Jira/GitHub Issue numbers (if any): \ DBOPS-429
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/database-devops/use-database-devops/deploying-database-schema.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Accelerate Innovation Velocity
+
+Harness DB DevOps enables organizations to accelerate their innovation velocity by automating the deployment of database changes. By integrating database changes into the CI/CD pipeline, teams can:
+
+   - **Ship Database Changes Like Application Code**: With DB DevOps, database changes can be treated similarly to application code, allowing for frequent and reliable updates without manual intervention. This reduces the time taken to deploy changes and minimizes the risk of errors associated with manual processes.
+   - **Visibility and Control**: The platform provides visibility into the state of database changes across environments, ensuring that teams can track what has been deployed and where. This transparency helps in identifying potential issues early in the deployment process.
+   - **Governance and Compliance**: Automated policies can be defined to ensure compliance with organizational standards, such as preventing destructive changes (e.g., dropping tables in production). This governance helps maintain stability while enabling rapid innovation.
+
+## Orchestration of Traditional Deployments
```
Comment (stephenatwell) [inline docs/database-devops/use-database-devops/deploying-database-schema.md:22]: we need to make it clear that this benefit only exists if using both DB DevOps AND Harness CD.

File: docs/database-devops/use-database-devops/deploying-database-schema.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   - **Rollback Capabilities**: In the event of a failure, the system supports smart rollbacks, allowing you to revert both application and database changes together. This minimizes downtime and ensures that the system can recover quickly from issues.
+   - **Integration with Existing Tools**: For organizations using other deployment tools, DB DevOps can be triggered externally, allowing for a seamless integration into existing workflows.
+
+## Deploying Database Schema Updates
+
+To deploy database schema updates using Harness DB DevOps, follow these steps:
+
+   1. Create a schema that includes a set of SQL scripts checked into a Git repository. This schema acts as a change log that will be referenced during the deployment process.
+   2. Set up a database connector that specifies the JDBC URL, username, and password for the database instance. This connector will be used to execute the schema updates.
+   3. Develop a pipeline within Harness that includes steps for deploying both application code and database changes. This pipeline should reference the schema defined in the previous step.
```
Comment (stephenatwell) [inline docs/database-devops/use-database-devops/deploying-database-schema.md:36]: m,issing step: creating a database instance to associate the db connector with the schema.

Comment (stephenatwell) [reply docs/database-devops/use-database-devops/deploying-database-schema.md:36]: the pipeline may or may not include steps for deploying application code depending on whether or not the customer also uses Harness CD.

File: docs/database-devops/use-database-devops/deploying-database-schema.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+## Orchestration of Traditional Deployments
+
+For companies using both Harness DB DevOps and Harness CD, both deployment types can be orchestrated in a unified pipeline that integrates both application and database changes. Key features include:
+
+   - **Single Unified Pipeline**: Harness allows you to create a single pipeline that encompasses both application and database deployments. This means that when a database change is made, it can be deployed alongside application updates, ensuring that both are in sync.
+   - **Rollback Capabilities**: In the event of a failure, the system supports smart rollbacks, allowing you to revert both application and database changes together. This minimizes downtime and ensures that the system can recover quickly from issues.
+   - **Integration with Existing Tools**: For organizations using other deployment tools, DB DevOps can be triggered externally, allowing for a seamless integration into existing workflows.
+
+## Deploying Database Schema Updates
```
Comment (archit-harness) [inline docs/database-devops/use-database-devops/deploying-database-schema.md:30]: @stephenatwell @jenniferopal  with each step should we add links to our docs how they can do it ? like creating schema we would have a doc which we can point? and same for other points?

Comment (jenniferopal) [reply docs/database-devops/use-database-devops/deploying-database-schema.md:30]: We can do that but I don't think docs on Integration tools within Database DevOps exists yet but I can incorporate it when it becomes available.

**Review by stephenatwell — APPROVED** [review]
lgtm

**Pull request merged**